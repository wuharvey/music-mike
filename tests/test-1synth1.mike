

pitch_list= p:[ 1 2 3 4 3 2 1];
mode=[1 3 5 6 8 10 12];
rhythm_list=r:[ q q q q q q q];
start_note=60;

s = Synth(pitch_list rhythm_list mode start_note 1  );
s1 = Synth(pitch_list rhythm_list mode 63 2  );
s2 = Merge(s s1);
Make_midi(s2 "plzwork.midi");
