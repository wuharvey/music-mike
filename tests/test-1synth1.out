[75]/0.10+ [68]/0.10 [65]/0.50 [70]/1.00+ [63]/1.00 [66]/0.05 [66]/1.00 [68]/0.05
